### ISSUE TITLE
[Feature Request] _van Hover function for DeepMD_

### ISSUE THREAD

**Issue by xuyangmark** [deepmodeling/deepmd-kit#2899]
### Summary

diffusion rate of ions is not only related to the activation energy but also to the ion jump frequency and jump radius, which requires the use of the van Hove function for analysis. Post-processing programs like VMD and ovito lack this functionality, it would be beneficial to incorporate it in deepmd-kit or AI4S Case Pavilion. 

### Detailed Description

The potential energy generated from the training of dpgen and deepmd-kit exhibits excellent generalization ability, which has greatly assisted our research on solid electrolytes. By fitting the mean square displacement (MSD), we can obtain the diffusion coefficient and activation energy of ions. However, I have encountered a problem where the modified material may have better diffusion coefficients at every temperature compared to the unmodified material, but the calculated activation energy is lower for the unmodified material. The possible reason behind these seemingly inconsistent conclusions is that the diffusion rate of ions is not only related to the activation energy but also to the ion jump frequency and jump radius, which requires the use of the van Hove function for analysis. This method is a powerful tool for molecular dynamics analysis and has been widely used in the following publications: (1) Chemistry of Materials, 2020, 32(19): 8373-8381; (2) Nat. Commun. 2017, 8, 15893; (3) Chemical Reviews 2020 120 (13), 5954-6008. Currently, it seems that post-processing programs such as VMD and ovito do not have the corresponding functionality. I am aware that the main purpose of deepmd-kit is to train potential functions for molecular dynamics simulation programs. However, it would be cool if this functionality could be added. In the two blog posts at https://bohrium.dp.tech/notebook/879b6a3a9d394448913ca1ce7de257ec and https://nb.bohrium.dp.tech/detail/7317436745, I felt the user-friendliness of these cases, which greatly reduced the threshold for researchers. Therefore, I highly recommend adding this functionality, either in deepmd-kit or in the AI4S Case Pavilion. 
The fundamental article for the principle of the van Hove function is Correlations in Space and Time and Born Approximation Scattering in Systems of Interacting Particles. Phys. Rev. 1954, 95, 249−262.

### Further Information, Files, and Links

(1)  Correlations in Space and Time and Born Approximation Scattering in Systems of Interacting Particles. Phys. Rev. 1954, 95, 249−262.  https://journals.aps.org/pr/abstract/10.1103/PhysRev.95.249
(2) Chemistry of Materials, 2020, 32(19): 8373-8381;  https://pubs.acs.org/doi/full/10.1021/acs.chemmater.0c02318
(2) Nat. Commun. 2017, 8, 15893;   https://www.nature.com/articles/ncomms15893
(3) Chemical Reviews 2020 120 (13), 5954-6008.  https://pubs.acs.org/doi/abs/10.1021/acs.chemrev.9b00747

**Comment by wanghan-iapcm** [comment]
Could you please check the bohrium link?  I cannot open it.

**Comment by njzjz** [comment]
I feel the proposal deviates from the primary purpose of the DeePMD-kit. DeePMD-kit is not a tool for post-processing or analyzing the simulation result. A standalone package is a better choice if anyone wants to contribute it.

**Comment by xuyangmark** [comment]
> Could you please check the bohrium link? I cannot open it.
Yes. The two websites are regarded as one link, thus could not be opened. Here it is:

https://bohrium.dp.tech/notebook/879b6a3a9d394448913ca1ce7de257ec

https://nb.bohrium.dp.tech/detail/7317436745

**Comment by xuyangmark** [comment]
> I feel the proposal deviates from the primary purpose of the DeePMD-kit. DeePMD-kit is not a tool for post-processing or analyzing the simulation result. A standalone package is a better choice if anyone wants to contribute it.

Yes, I agree. Thanks

**Comment by xuyangmark** [comment]
> Could you please check the bohrium link? I cannot open it.

I agree with njzjz. It might be more suitable to put it to cases of bohrium notebook. I just don't know how to find the right person to ask for help or cooperate with. I'd be very gladly to add the cooperator as co-corresponding author in the paper.  It could hard be published at journals such as J. Am. Chem. Soc., but could be published at Journal of Physical Chemistry C or Physical Chemistry Chemical Physics.

**Comment by njzjz** [comment]
I converted it to a discussion since it is not a feature request of the DeePMD-kit itself.
